universal-application-tool-0.0.1/app/models/Question.java

```diff
@@ -63,14 +69,16 @@ public void persistChangesToQuestionDefinition() {
   @PostUpdate
   public void loadQuestionDefinition() {
     this.questionDefinition =
-        new QuestionDefinition(
-            this.id,
-            this.version,
-            this.name,
-            this.path,
-            this.description,
-            this.questionText,
-            Optional.ofNullable(this.questionHelpText));
+        new QuestionDefinitionBuilder()
+            .setId(this.id)
+            .setVersion(this.version)
+            .setName(this.name)
+            .setPath(this.path)
+            .setDescription(this.description)
+            .setQuestionText(this.questionText)
+            .setQuestionHelpText(Optional.ofNullable(this.questionHelpText))
+            .setQuestionType(QuestionType.valueOf(questionType))
```

Comment:
nit: Remove `this` from the above variables (or add it to `questionType`) for consistency. Prefer to remove `this` for the reason bion mentioned elsewhere.